samples/subsys/mgmt/osdp/control_panel/sample.yaml

```diff
@@ -4,7 +4,7 @@ sample:
 tests:
   sample.mgmt.osdp.control_panel:
     tags: osdp
-    depends_on: gpio
+    depends_on: gpio arduino_serial
```

Comment:
This test now depends on `arduino_serial`, therefore can't use stm32_min_dev_black as an integration platform.
Please use a different board.

Reply:
Switched to `nucleo_g474re` at both places. Thanks!